src/main/java/org/truffleruby/language/TruffleBootNodes.java

```diff
@@ -109,13 +109,14 @@ private Object loadMain(IndirectCallNode callNode, String inputFile) {
                     null);
 
             final CodeLoader.DeferredCall deferredCall = getContext().getCodeLoader().prepareExecute(
-                    ParserContext.TOP_LEVEL,
+                    ParserContext.TOP_LEVEL_FIRST,
```

Comment:
It's just for consistency with the line above. Both values have no effect for `prepareExecute()`.